docs/repositories-configure/managing-branches.md

```diff
@@ -15,7 +15,7 @@ To change the default branch of your repository or enable analysis on other bran
 
 1.  To change the default branch on Codacy, click the corresponding **Make default** link that appears when you hover the column **Default**.
 
-    Codacy displays information for the default branch on dashboards and the repositories list by default, and we recommend that you set the default branch on Codacy to match the default branch of your Git repository.
+    Codacy displays information for the default branch on dashboards. Consider matching the default branch on Codacy to the default branch on your Git provider.
```

Comment:
Isn't this recommendation a bit hidden down here? Does it make sense to move it to the top? (for example, inside an infobox after the first sentence)

Reply:
Indeed, this should be part of the "concept" instead of the "task". :nerd_face:

Reply:
I moved the sentence to an infobox as suggested, although it reads a bit weird...

Reply:
Looks good to me. 👍